sycl/test/plugins/lspi_gpu_cuda.cpp

```diff
@@ -0,0 +1,24 @@
+// REQUIRES: gpu, cuda
```

Comment:
Isn't the test will run ONLY on CUDA BE?

Reply:
I think you are right, reworked.